### ISSUE TITLE
LightNER Implementation for Other Languages

### ISSUE THREAD

**Issue by khairunnisaor** [zjunlp/DeepKE#169]
**Describe the question**
> A clear and concise description of what the question is.
Hi, 
thank you for your amazing toolkit for entity-related tasks! I am interested to implement the few-shot NER for other languages and need to use the mBART model. However, it uses different tokenizers (MBart50Tokenizer), but it is not supported by DeepKE.

Do you have any advice related to this issue?

Thank you in advance!

**Environment (please complete the following information):**
 - OS: mac
 - Python 3.8

**Screenshots**
> If applicable, add screenshots to help explain your problem.

**Additional context**
> Add any other context about the problem here.

**Comment by zxlzr** [comment]
That's a good question and we will plan to add the support of different tokenizers soon.

**Comment by khairunnisaor** [comment]
Is it sometime around this month? 

Anyway, I tried to include the different tokenizers manually but got this error.
```OSError: Model name 'facebook/mbart-large-cc25' was not found in tokenizers model name list (facebook/bart-base, facebook/bart-large, facebook/bart-large-mnli, facebook/bart-large-cnn, facebook/bart-large-xsum, yjernite/bart_eli5). We assumed 'facebook/mbart-large-cc25' was a path, a model identifier, or url to a directory containing vocabulary files named ['vocab.json', 'merges.txt'] but couldn't find such vocabulary files at this path or url.```

Looks like the mentioned models are stored somewhere by default on the code (or cache, I'm not sure). However, I couldn't find where it is.. do you have any idea about this?

Thank you!

**Comment by pengzju** [comment]
Your error message indicates that ['vocab.json', 'merges.txt'] could not be found. You can try to use **MBartTokenizer**, more details can be found at 
https://huggingface.co/docs/transformers/main/model_doc/mbart
